Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml

```diff
@@ -188,44 +188,68 @@
+    <string name="validation_rules_result_cannot_be_checked_result_title">"Ihr Zertifikat konnte nicht vollständig geprüft werden"</string>
     <!-- XTXT: Validation Rule Result Not Valid - Result  -->
-    <string name="validation_rules_result_not_valid_result_title">Ihr Zertifikat ist im gewählten Land nicht gültig</string>
+    <string name="validation_rules_result_not_valid_result_title">"Ihr Zertifikat ist im gewählten Land nicht gültig"</string>
 
     <!-- XTXT: Validation Rule Open Header Title  -->
-    <string name="validation_rules_open_header_title">Prüfung nicht möglich</string>
+    <string name="validation_rules_open_header_title">"Prüfung nicht möglich"</string>
     <!-- XTXT: Validation Rule Open Header Subtitle  -->
-    <string name="validation_rules_open_header_subtitle">Nachfolgende Regeln konnten nicht geprüft werden. Vergewissern Sie sich, dass das Zertifikat den Regeln Ihres Reiselandes entspricht.</string>
+    <string name="validation_rules_open_header_subtitle">"Nachfolgende Regeln konnten nicht geprüft werden. Vergewissern Sie sich, dass das Zertifikat den Regeln Ihres Reiselandes entspricht."</string>
     <!-- XTXT: Validation Rule Failure Header Title  -->
-    <string name="validation_rules_failure_header_title">Prüfung abgeschlossen</string>
+    <string name="validation_rules_failure_header_title">"Prüfung abgeschlossen"</string>
     <!-- XTXT: Validation Rule Failure Header Subtitle  -->
-    <string name="validation_rules_failure_header_subtitle">Folgende Regeln sind nicht erfüllt:</string>
+    <string name="validation_rules_failure_header_subtitle">"Folgende Regeln sind nicht erfüllt:"</string>
     <!-- XTXT: Validation Rule Technical Failure Header Title  -->
-    <string name="validation_rules_technical_failure_header_title">Technische Prüfung</string>
+    <string name="validation_rules_technical_failure_header_title">"Technische Prüfung"</string>
     <!-- XTXT: Validation Rule Technical Failure Header Subtitle  -->
-    <string name="validation_rules_technical_failure_header_subtitle">Folgende Regeln sind nicht erfüllt:</string>
+    <string name="validation_rules_technical_failure_header_subtitle">"Folgende Regeln sind nicht erfüllt:"</string>
     <!-- XTXT: Validation Rule Technical Failure - Date Format  -->
-    <string name="validation_rule_technical_error_date_format">Das Zertifikat muss dem erwarteten Datenformat entsprechen.</string>
+    <string name="validation_rule_technical_error_date_format">"Das Zertifikat muss dem erwarteten Datenformat entsprechen."</string>
     <!-- XTXT: Validation Rule Technical Failure - Date Expired  -->
-    <string name="validation_rule_technical_error_date_expired">Das Zertifikat darf nicht abgelaufen sein.</string>
+    <string name="validation_rule_technical_error_date_expired">"Das Zertifikat darf nicht abgelaufen sein."</string>
```

Comment:
This message needs to be replaced by the new message "Das technische Ablaufdatum darf nicht überschritten sein.
" as described in Figma: https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.2?node-id=25537%3A21835